slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/interchange/model/Metadata.java

```diff
@@ -31,6 +32,7 @@ public Metadata(
     this.genesisValidatorsRoot = genesisValidatorsRoot;
   }
 
+  @JsonIgnore
```

Comment:
Looks like the getFormatVersionAsString is probably unnecessary and this this could be getter instead.

Reply:
In that getFormatVersionAsString is only used in the tests. The value used in production code is coming from this function

Reply:
get version as string is what is used by the exporter - and it is required as the standard states that the version is a STRING not a numeric value :(